.github/workflows/ci.yml

```diff
@@ -17,39 +21,55 @@ jobs:
   build:
     if: github.event.pull_request.draft == false
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest]
         python-version: [ '3.6', '3.7', '3.8', '3.9' ]
+        include: # Run macos and windows tests on python 3.9 only
+          - os: windows-latest 
+            python-version:  '3.9'
+          - os: macos-latest 
+            python-version:  '3.9'
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+
       - name: Install dependencies
         run: |
-          sudo apt-get install build-essential
-          pip install --upgrade pip
-          pip install -r requirements/dev.txt
-          pip install -e .[prophet]
-          pip install -e .[torch]
-          pip freeze
+          # sudo apt-get install build-essential # build-essential appears to already be up-to date on all latest github os'. Nice to remove as otherwise need conditional step dependent on os
+          python -m pip install --upgrade pip
+          python -m pip install wheel # To avoid having to install dependencies from setup.py's
+          python -m pip install -r requirements/dev.txt
+          if [ "$RUNNER_OS" != "Windows" ]; then  # Skip Prophet tests on windows for now (need to ensure mingw-w64 compiler toolchain installed properly)
```

Comment:
`OutlierProphet` tests are skipped on Windows for now as it was proving difficult to install cleanly (due to need to install `pystan`).